Pod/Classes/Playback/internal/_WistiaPlayer.swift

```diff
@@ -92,6 +92,10 @@ internal extension WistiaPlayer {
     //
     //We just need HLS (if required), otherwise mp4.  If there are options, we pick the best sized.
     internal func bestPlaybackUrl(for media: WistiaMedia, andAssetWithSlug assetSlug: String?, requiringHLS requireHLS: Bool, atTargetWidth targetWidth: Int64) throws -> URL {
+        let delegateUrl = delegate?.wistiaPlayer(self, shouldLoadVideoForMedia: media)
+        guard delegateUrl == nil else {
+            return delegateUrl!
```

Comment:
need to call `delegate?.wistiaPlayer(self, willLoadVideoForMedia: media, usingAsset: nil, usingHLSMasterIndexManifest: false)` before returning